src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -1565,16 +1585,18 @@ def append_mmmc(cmd: str) -> None:
                 name="{n}.hold_cond".format(n=hold_corner.name),
                 list="{n}.hold_set".format(n=hold_corner.name)
             ))
-            # Next, create Innovus rc corners from qrc tech files
-            append_mmmc("create_rc_corner -name {name} -temperature {tempInCelsius} {qrc}".format(
+            # Next, create Innovus rc corners from qrc tech or cap table files 
+            append_mmmc("create_rc_corner -name {name} -temperature {tempInCelsius} {qrc} {cap_table}".format(
                 name="{n}.setup_rc".format(n=setup_corner.name),
                 tempInCelsius=str(setup_corner.temp.value),
-                qrc="-qrc_tech {}".format(self.get_mmmc_qrc(setup_corner)) if self.get_mmmc_qrc(setup_corner) != '' else ''
+                qrc="-qrc_tech {}".format(self.get_mmmc_qrc(setup_corner)) if self.get_mmmc_qrc(setup_corner) != '' else '',
```

Comment:
I'm sure you tested this, but does `-qrc_tech` and `-cap_table` options need to be exclusive? Also, the if/else on these lines seem redundant.

Reply:
They do not need to be exclusive. The tool will gladly accept both and will simply use the qrc tech file.

Reply:
If/else either return "-qrc_tech {file}" or an empty string if the {file} is an empty string. So unless I'm missing something I don't believe it's redundant

Reply:
I see now!